### PR TITLE
release v1.3.2

### PR DESCRIPTION
**Pull request by ElRodrigote**
* Added new token logos for CoW, Shutter, and Olas
* Added Shutter and Olas to our Gnosis token list
* Created env vars to handle our subgraph API endpoints
* Bugfixes:
  * for singe-slot and no-slot orders:
    * Make these orders cancellable under the correct scenarios
    * My stacks page was breaking the app
    * My Stacks were displaying `NaN` for average buy prices
  * Stackbox:
    * Now it hides the stack details tile if the end date is in the past.
    * Enabled the mobile users to place decimal amounts in the Amount input
    * Allowed to create stacks with a start date close to the current timestamp of the user
    * Updates the past end date error when the end date changes
  * Stack Details:
    * Now displays the correct orders progress bar.
    * Allows stack funds withdrawal for some edge cases where the stacks only spent the protocol fee
  * The Network Switch component was not picking up the right chain ID in some scenarios
  * My Stacks now updates correctly on network switch

### CODE REVIEW

**Review by berteotti — APPROVED** [review]
Good job Rorry!

File: packages/app/components/stackbox/Stackbox.tsx [hunk 1]
```diff
@@ -486,6 +488,10 @@ export const Stackbox = () => {
                   <DatePicker
                     dateTime={endDateTime}
                     setDateTime={(date: Date) => {
+                      const isEndTimeBeforeStartTime =
+                        date.getTime() <= startDateTime.getTime();
+
+                      setShowPastEndDateError(isEndTimeBeforeStartTime);
```
Comment (Diogomartf) [inline packages/app/components/stackbox/Stackbox.tsx:494]: shouldn't we only set if `isEndTimeBeforeStartTime` is `true`?

Comment (ElRodrigote) [reply packages/app/components/stackbox/Stackbox.tsx:494]: If we don't set the `false` value the error won't get cleaned on a valid date change

**Review by Diogomartf — APPROVED** [review]
Great job on getting this release Rorry out!

It sure killed a lot of bugs.

**Pull request merged**